### ISSUE TITLE
Make svg a default output format

### ISSUE THREAD

**Issue by KOLANICH** [ipython/ipykernel#251]

**Comment by Carreau** [comment]
I'm unsure what your question/suggestion is. If an object has an SVG representation it is sent to the frontend. There is no need to make it "default".

**Comment by KOLANICH** [comment]
I mean that every my notebook I have to start with
```python
%config InlineBackend.figure_format = 'svg'
```

I wonder why matplotlib output is not set by default to ```svg``` in jupyter notebook setting taking in accounts the facts that jupyter notebook uses web browsers for GUI, all modern web browsers support svg and svg is a vector format, which is good for drawing curves and schemas, such as matplotlib plots.

**Comment by takluyver** [comment]
SVG tends to get horrible when you have very large numbers of elements, IIRC. E.g. mpl can handle a scatter plot of millions of points, but nobody wants the result as an SVG.

**Comment by KOLANICH** [comment]
I guess then it'll require some heuristics and matplotlib monkey-patching to choose the optimal format automatically then.

**Comment by takluyver** [comment]
I don't think this is something where the tools should try to be 'smart'. Simplicity and predictability are good things too. :-)
